include/collection.impl.hh

```diff
@@ -108,7 +114,7 @@ namespace cmk {
     // tree builder... the code there is better commented for the time being:
     // https://github.com/jszaday/hypercomm/blob/main/include/hypercomm/tree_builder/tree_builder.hpp
     // TODO ( copy the comments from there)
-    template <typename T, template <class> class Mapper>
+    template <typename T, template <class> class Mapper, typename IsGroup>
```

Comment:
```suggestion
    template <typename T, template <class> class Mapper>
```

Reply:
I am unsure whether omitting this is more visually pleasing or not; however, it's fine either way. Omitting it would require "explicitly" specializing the class so it's probably unnecessary.

Reply:
Agree that omitting this looks better. I added it to avoid 2 separate specializations for `nodegroup_mapper` and `group_mapper`. Is that possible without having the `Enable` parameter?

Reply:
I believe that you can rewrite it like [this](https://ideone.com/vjgkDm):

```cpp
template <typename T, template <class> class Mapper>
class collection_bridge_<T, Mapper> : public collection_base_ {
...
```